user/tests/unit/makefile

```diff
@@ -65,7 +69,19 @@ INCLUDE_DIRS += $(HAL)inc
 INCLUDE_DIRS += $(COMMUNICATION)src
 INCLUDE_DIRS += dynalib/inc
 
-CFLAGS += $(patsubst %,-I$(SRC_ROOT)%,$(INCLUDE_DIRS)) -I.
+# prefix $(SRC_ROOT)
+ABS_INCLUDE_DIRS += $(patsubst %,$(SRC_ROOT)/%,$(INCLUDE_DIRS)) 
+
+
+ifeq ("$(BOOST_ROOT)","")
+$(error BOOST_ROOT not defined)
```

Comment:
The unit tests no longer run because of this. I suppose I need to install Boost. Can you tell me how you set it up so I don't end up with a different setup than yours?